analyses/README.md

```diff
@@ -16,6 +16,29 @@ docker exec -ti pathogenecity_anno bash
 
 ## How to Run
 
+### Pre-requisite
+1. Download ClinVar database
+bash download_db_files.sh
+```
+If specific date of ClinVar dataset is needed use archive site and download clinvar, submission_summary and variant_summary
+Example: 
+wget ftp.ncbi.nlm.nih.gov/pub/clinvar/vcf_GRCh38/archive_2.0/2023/clinvar_20230218.vcf.gz
+wget ftp.ncbi.nlm.nih.gov/pub/clinvar/tab_delimited/archive/variant_summary_2023-02.txt.gz
+wget ftp.ncbi.nlm.nih.gov/pub/clinvar/tab_delimited/archive/submission_summary_2023-03.txt.gz
+```
+Unzip variant summary and submission summary files
+```
+
+2. VEP annotated VCF
+```
+example:
+vep --offline --cache --dir_cache $VEP_CACHEDIR --fasta $VEP_CACHEDIR/GRCh38.fa --refseq --use_given_ref --species homo_sapiens --assembly GRCh38 --fork 1 --hgvs --hgvsg --canonical --symbol --distance 0 --exclude_predicted --flag_pick --lookup_ref --force --input_file testing_010423.vcf --output_file testing_010423_VEP.vcf --format vcf --vcf --no_stats --numbers
+```
+3. InterVar annotation 
+https://github.com/WGLab/InterVar
+4. AutoPVS1 annotation
+https://github.com/JiguangPeng/autopvs1
+
```

Comment:
Is there a way we can tie this to a specific version or make sure as we develop, the versions don't give us any surprises in the code downstream? Eg - simple column name updates etc - what versions of each tool is this being tested on ?

Reply:
that is actually a good point. VEP does get updated quite regularly so that might be important... I don't believe between different versions scripts are changing, but output can... This is worth noting in the README file